README.md

```diff
@@ -60,115 +48,38 @@
 
-## API Keys
+### Input and Output file storage
 
-Also known as the `client_id`. You must additionally pass in your Adobe API key in the `x-api-key` header field. You’ll automatically get a developer API key when you create your Adobe I/O Console Integration.  After you've created your integration you can find your API key in the `Overview` tab of your Integration
+Clients can use assets stored on one of the following storage types:
+1. Adobe: by referencing the path to the files on Creative Cloud
+2. External: (like AWS S3) by using a presigned GET/PUT URL
+3. Azure: By generating a SAS (Shared Access Signature) for upload/download
+4. Dropbox: Generate temporary upload/download links using https://dropbox.github.io/dropbox-api-v2-explorer/
 
-## Retries
 
-- The service will retry status codes of 429, 502, 503, 504 three times.
-- You should only retry requests that have a 5xx response code. A 5xx error response indicates there was a problem processing the request on the server.
-- You should implement an exponential back-off retry strategy with 3 retry attempts.
-- You should not retry requests for any other response code.
+### SmartObject
 
-## Rate Limiting
+The Photoshop APIs currently support creating and editing of Embedded Smart Objects. Support for Linked Smart Objects is forthcoming.
 
-We have not put a throttle limit on requests to the API at this time.
+- In order to update an embedded smart object that is referenced by multiple layers you only need to update one of those layers, the effect will be reflected in all layers referencing the same smart object.
 
-# Photoshop
+- The replaced smart object takes the bounds of the new image by default. If your document contains transparent pixels (e.g some .png) , you may not get consistent bounds.
 
-## General Workflow
+The API's are documented [here](https://adobedocs.github.io/photoshop-api-docs/#api-Photoshop-document_operations)
 
-The typical workflow involves retrieving a PSD document manifest file via `/documentManifest` (a JSON representation of the documents layer tree), followed by one or more calls to `/documentOperations` to optionally edit the PSD and/or create new image renditions. Both endpoints are asynchronous so the response will contain the `/status` endpoint to poll for job status and results
+We also have an example of replacing a Smart Object within a layer.
 
-### Input and Output file storage
+[Smart Object Example Code](https://github.com/AdobeDocs/photoshop-api-docs#example-6-swapping-the-image-in-a-smart-object-layer)
 
-Clients can use assets stored on one of the following storage types:
-1. Adobe: by referencing the path to the files on Creative Cloud
-2. External: (like AWS S3) by using a presigned GET/PUT URL
-3. Azure: By generating a SAS (Shared Access Signature) for upload/download
-4. Dropbox: Generate temporary upload/download links using https://dropbox.github.io/dropbox-api-v2-explorer/
+Smart Object support is a work in progress.
```

Comment:
What do we mean by to convey to clients by the statement "is a work in progress"? 

I suggest instead we state what we know works, and what we know doesn't work.

Reply:
@alshamma Sure. Will make these changes once i gather all the review points.